src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -142,6 +144,7 @@ export default observer(function GamePage(): JSX.Element | null {
   const notSupportedGame =
     gameInfo.runner !== 'sideload' && gameInfo.thirdPartyManagedApp === 'Origin'
   const isOffline = connectivity.status !== 'online'
+  const isBrowserGame = Boolean(gameInfo.browserUrl)
```

Comment:
```suggestion
    const isBrowserGame =
    installPlatform === 'Browser' || installPlatform === 'web'
```

^ copied from game card https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/blob/d8a155da6e66957c6051a796ab6a05cbd157c47c/src/frontend/screens/Library/components/GameCard/index.tsx#L167

I believe we just had an issue recently with this flag since some game infos have a browserUrl but are not installed browser games. I believe this happens when a game has a native and browser platform but the native platform is installed. This would still return true.

Reply:
yes you made that change on this pr 3 weeks ago https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/pull/1054

Reply:
Yeah, was to fix that issue with the game card.
I think the problem here is that sometimes we use 'Browser' and sometimes 'web'  and now 'webgl' so this is getting confusing and needing this type of workaround. 
Ideally we should receive the `isBrowserGame`  (or build) from the API so we would use it for everything and we could handle what is really a browser game from there.

Reply:
Well `webgl` isn't for a web game. It is just a one-off for the Unfettered team really. The api just supplies a map of platform keys to platform meta for a release and it's up to the client to decide which platform to install, whether that be browser or native. 

I think the suggested change above works `installPlatform === 'Browser' || installPlatform === 'web'`. I think `"Browser"` is just for backwards compatibility? `web` is the platform key for a browser game in a release currently